package.json

```diff
@@ -37,13 +37,16 @@
         "lint:fix:md": "markdownlint --fix  \"**/*.md\"",
         "lint:scss": "stylelint \"**/*.scss\"",
         "lint:fix:scss": "stylelint \"**/*.scss\" --fix",
+        "format": "prettier --write \"**/*.+(js|json|scss)\"",
+        "format:check": "prettier --check \"**/*.+(js|json|scss)\"",
         "start": "npm-run-all build:sass --parallel *:*:watch"
     },
     "lint-staged": {
         "**/*.{js,jsx}": "eslint --fix",
         "*.md": "markdownlint --fix",
         "*.scss": "stylelint \"**/*.scss\" --fix",
-        "**/*.svg": "npx svgo -r --multipass"
+        "**/*.svg": "npx svgo -r --multipass",
+        "**/*.{js,jsx,scss}": "prettier --write"
     },
```

Comment:
Per https://github.com/lint-staged/lint-staged?tab=readme-ov-file#task-concurrency, if multiple commands modify the same files, the commands should be run in order.

Like in the eslint/rewrite repo:

https://github.com/eslint/rewrite/blob/66a97f9ebb09ec29a2b12c05fbd215b18363f755/package.json#L22-L28